app/components/Gallery.js

```diff
@@ -60,7 +69,15 @@ export default class Gallery extends Component {
               icon={<Icon name='code' color='#ffffff' />}
               backgroundColor='#03A9F4'
               buttonStyle={{borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
-              title='VIEW NOW' />
+              title='Learn In Augmented Reality' />
+
+              <Text></Text>
+
+              <Button
+                icon={<Icon name='code' color='#ffffff' />}
+                backgroundColor='#ffb300'
+                buttonStyle={{borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
+                title='Learn In Virtual Reality' />
```

Comment:
Things like this title ("Learn In Virtual Reality") get repeated a lot. To avoid repeating yourself, and so that you can change all the titles in one place, store this string in a variable somewhere.